### PR TITLE
Properly handle snapshots if camera added to bridge.

### PR DESCRIPTION
**Pull request by bzzeke**
The `handle_resource` method from `HAPServer` class does not support situation when camera added to bridge and fails with `Got a request for snapshot, but the Accessory does not define a "get_snapshot" method` error. This is because it tries to check if bridge itself has the `get_snapshot` method.

This pull request fixes this problem.

### CODE REVIEW

File: pyhap/hap_server.py [hunk 1]
```diff
@@ -664,13 +664,23 @@ def _handle_remove_pairing(self, tlv_objects):
 
     def handle_resource(self):
         """Get a snapshot from the camera."""
-        if not hasattr(self.accessory_handler.accessory, 'get_snapshot'):
+
+        data_len = int(self.headers["Content-Length"])
+        request_body = self.rfile.read(data_len)
+        data = json.loads(request_body.decode("utf-8"))
+
+        if self.accessory_handler.accessory.category == CATEGORY_BRIDGE:
```
Comment (bdraco) [inline pyhap/hap_server.py:672]: Can we check for `STANDALONE_AID` instead?

Comment (bzzeke) [reply pyhap/hap_server.py:672]: Just tried, but `self.accessory_handler.accessory.aid` returns `1` both for bridge and standalone accessory

Comment (bzzeke) [reply pyhap/hap_server.py:672]: I've created new pull request which merges correctly with 3.2.0.
https://github.com/ikalchev/HAP-python/pull/306

**Comment by bdraco** [comment]
I think this can be closed in favor of https://github.com/ikalchev/HAP-python/pull/306

**Review by bdraco — CHANGES_REQUESTED** [review]
I think this can be closed in favor of https://github.com/ikalchev/HAP-python/pull/306

**Pull request closed without merging**